smart-contracts/scripts/bulkSetTokenLockBurnLimit.js

```diff
@@ -63,7 +63,7 @@ module.exports = async (cb) => {
     try {
         const accounts = await web3.eth.getAccounts();
 
-        bridgeBank = await BridgeBank.deployed()
+        bridgeBank = await BridgeBank.at(process.env.ADDRESS)
         console.log(await bridgeBank.bulkWhitelistUpdateLimits(addresses, limits, {
             from: accounts[0],
             gas: 300000 // 300,000 gas
```

Comment:
Do we need a higher limit on ropsten or mainnet?

Reply:
We can bump that up if needed when sending huge whitelist requests.